internal/commands/commands.go

```diff
@@ -77,6 +77,10 @@ func multiValueHelp(name string) string {
 	return fmt.Sprintf("\nRepeat for each %s in order,\n  or supply once by comma-separated list", name)
 }
 
+func prependExperimental(short string) string {
+	return fmt.Sprintf("%s: %s", style.Symbol("Experimental"), short)
```

Comment:
I love this feature as a whole. I would suggest a few tweaks to the visual output.

1. I would use yellow or no color.
2. Instead of `Experimental:` it may look better as `(experimental)` given it's surroundings.

Reply:
Yellow, just to make the contrast higher?

Reply:
`Symbol` in my mind is used for "values" where this to me feels more like an annotation that should call attention similar to a warning message.

Reply:
![image](https://user-images.githubusercontent.com/7035673/94187281-675bf380-feb0-11ea-92b2-192d8cec85d4.png)

Thoughts?